iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractConnectableXml.java

```diff
@@ -153,31 +151,39 @@ public static void readCurrentLimits(Integer index, Supplier<CurrentLimitsAdder>
                 double value = XmlUtil.readOptionalDoubleAttribute(reader, "value", Double.MAX_VALUE);
                 boolean fictitious = XmlUtil.readOptionalBoolAttribute(reader, "fictitious", false);
                 adder.beginTemporaryLimit()
-                    .setName(name)
-                    .setAcceptableDuration(acceptableDuration)
-                    .setValue(value)
-                    .setFictitious(fictitious)
-                    .endTemporaryLimit();
+                        .setName(name)
+                        .setAcceptableDuration(acceptableDuration)
+                        .setValue(value)
+                        .setFictitious(fictitious)
+                        .endTemporaryLimit();
             }
         });
         adder.add();
     }
 
     public static void writeCurrentLimits(Integer index, CurrentLimits limits, XMLStreamWriter writer) throws XMLStreamException {
```

Comment:
To be discussed: I fact, I have a doubt now... This method is not really public, but private package... maybe you were right yesterday when I asked you to remove the deprecated. I think we should even break the API and make this method private package.

Is this method still used?

Reply:
Currently, it is only used in some extensions relying on `CurrentLimits`. Not sure either if this should remain public or not but I think both positions are valid.